README.md

```diff
@@ -317,20 +317,82 @@ sct_run_batch -jobs 32 -path-data ~/path-to-dataset/epi-stanford/ \
 ~~~
 
 
-## 8. Active learning procedure (TODO)
+## 8. Using contrast-agnostic model with nnUNet
+
+This section provides instructions on how to train nnUNet models with pre-trained weights of the contrast-agnostic model. Ideal use cases include using the 
+pretrained weights for training/finetuning on any spinal-cord-related segmentation task (e.g. lesions, rootlets, etc.). 
+
+### Step 1: Download the pretrained weights
+
+Download the pretrained weights from the [latest release](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases) of the 
+contrast-agnostic model. This refers to the `.zip` file of the format `model_contrast_agnostic_<date>_nnunet_compatible.zip`. 
+
+> [!WARNING]  
+> Only download the model with the `nnunet_compatible` suffix. If a release does not have this suffix, then that model weights are not directly 
+compatible with nnUNet.
+
+
+### Step 2: Create a new plans file
+
+1. Create a copy of the original `nnUNetPlans.json` file (found under `$nnUNet_preprocessed/<dataset_name_or_id>`) and 
+rename it to `nnUNetPlans_contrast_agnostic.json`. 
+2. In the `nnUNetPlans_contrast_agnostic.json`, modify the values of the following keys in the `3d_fullres` dict to be able to match the 
+values used for the contrast-agnostic model:
+
+```json
+
+"patch_size": [64, 192, 320],
+"n_stages": 6,
+"features_per_stage": [32, 64, 128, 256, 320, 320],
+"n_conv_per_stage": [2, 2, 2, 2, 2, 2],
+"n_conv_per_stage_decoder": [2, 2, 2, 2, 2],
+"strides": [
+    [1, 1, 1],
+    [2, 2, 2],
+    [2, 2, 2],
+    [2, 2, 2],
+    [2, 2, 2],
+    [1, 2, 2]
+    ],
+"kernel_sizes":[
+    [3, 3, 3],
+    [3, 3, 3],
+    [3, 3, 3],
+    [3, 3, 3],
+    [3, 3, 3],
+    [3, 3, 3]
+    ],
+"strides": [
+    [1, 1, 1],
+    [2, 2, 2],
+    [2, 2, 2],
+    [2, 2, 2],
+    [2, 2, 2],
+    [1, 2, 2]
+    ],
 
-> **Note**
-> This section is still a work in progress.
+```
+
+### Step 3: Train/Finetune the nnUNet model on your task
+
+Provide the path to the downloaded pretrained weights:
 
-To extend the training set to other contrasts and to pathologies, we applied the segmentation model to other datasets, manually corrected the segmentations and added them to the training set.
+```bash
+
+nnUNetv2_train <dataset_name_or_id> <configuration> <fold> -p nnUNetPlans_contrast_agnostic.json -pretrained_weights <path_to_pretrained_weights> -tr <nnUNetTrainer_Xepochs>
```

Comment:
the command is working for me without the `json` suffix (I have not tested with the sufix)
```suggestion
nnUNetv2_train <dataset_name_or_id> <configuration> <fold> -p nnUNetPlans_contrast_agnostic -pretrained_weights <path_to_pretrained_weights> -tr <nnUNetTrainer_Xepochs>
```